Addons/tsgExoticQuadrature.hpp

```diff
@@ -130,29 +151,32 @@ inline double lagrange_eval(size_t idx, const std::vector<double> &roots, double
  * (weight_fn_vals[i] + shift) for every i.
  * \endinternal
  */
-inline TasGrid::CustomTabulated getShiftedExoticQuadrature(const int n, const double shift,
+inline TasGrid::CustomTabulated getShiftedExoticQuadrature(const int n, const double shift, const std::vector<double> &weight_fn_vals,
                                                            const std::vector<double> &shifted_weights, const std::vector<double> &ref_points,
                                                            const char* description, const bool is_symmetric = false) {
-
-    // Create the set of points (roots) for the first term.
+    #ifndef Tasmanian_ENABLE_BLAS
+    if (gauss_quadrature_version == 1) {
+        #ifndef NDEBUG
+        std::cout << "WARNING: BLAS acceleration is not enabled!\nSwitching to an exotic quadrature algorithm that does not"
```

Comment:
This is way too much info to dump on the user. You can use it for yourself, but will not merge it into master.

We should not be generating a warning for something that we support and should work just fine. What is even worse, we are saying "Success!" right after the warning. Why would the user care and what difference does it make.

Reply:
Gotcha. I removed the messages and moved this into `getGaussNodesAndWeights()` since that is where it is needed.